home/static/data/user_data.json

```diff
@@ -0,0 +1,362 @@
+[
+  {
```

Comment:
this is really sensitive data, you cannot push to git public like this
Everything relate to database should be careful